### PR TITLE
[16.0][FIX] lighting_reporting: datasheet fixes and improvements

### PR DESCRIPTION
**Pull request by FrankC013**

### CODE REVIEW

**Review by eantones — CHANGES_REQUESTED** [review]
Rename the commits from "datasheet fixes and improvements" to something like "New layout" in lighting_reporting and lighting_reporting_seo.
The commit should be "IMP" if there are fixes they should go to another commit

File: lighting_reporting/static/src/scss/styles.scss [hunk 1]
```diff
@@ -242,21 +218,6 @@ div.footer {
...
-        font-family: "Raleway";
-        font-size: 10px; // 7pt
-        line-height: 16px; // 10pt
-        text-align: justify;
-        text-justify: inter-word;
-        border: 0;
-        padding: 0;
-    }
-}
-
```
Comment (eantones) [inline lighting_reporting/static/src/scss/styles.scss:259]: Why did you remove this?

**Review by eantones — CHANGES_REQUESTED** [review]

File: lighting_reporting/static/src/scss/styles.scss (hunk 1 above)
Comment (FrankC013) [reply lighting_reporting/static/src/scss/styles.scss:259]: This class was only used by the lighting_reporting_seo module, and I have moved it

**Review by eantones — APPROVED** [review]
